core/contracts/oracle/implementation/InterfaceRegistry.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.6.0;
+
+
+/**
+ * @title Stores common interface names used through the protocol through registration in the Finder.
+ */
+library InterfaceRegistry {
```

Comment:
I think this is a great solution. Also +1 on not entangling it with the Finder in any way since this will be a "living list" that may change in the future as the system is upgraded.